spec/rubocop/compeon_spec.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.describe Rubocop::Compeon do
```

Comment:
[Offtopic] Do we really want to keep the boilerplate?

Reply:
For the case that we want to add custom cops.